packages/redux-dynamic-modules-react/src/DynamicModuleLoader.tsx

```diff
@@ -12,6 +12,9 @@ export interface IDynamicModuleLoaderProps {
     /** Modules that need to be dynamically registerd */
     modules: IModuleTuple;
 
+    // boolean flag to determine if dynamically loaded modules should be automatically unloaded when component is dismounted

```

Comment:
As I think more I think this property should be on the module itself. Because semantically we want a module to be always persisted instead of the component loading it deciding weather to keep it or not. That way this feature will extend to non-react implementations as well, and the recent refactor allows that use case.

Reply:
I agree here, this is more a concern of the module, because it knows whether or not the area it deals with is persistent or not

Reply:
Yes, that makes sense.  It would be nice if you could specify this as an option in the module configuration object

Reply:
@ahrnee do you plan to make the change or want me to take a stab?

Reply:
Please see #89

Reply:
When will this change merged to release?

Reply:
Already released.